cmd/ontap-cvo.go

```diff
@@ -652,6 +664,33 @@ func formatSharesMap(shares map[string][]string) map[string]string {
 	return result
 }
 
+// This will check for duplicate shares
+func validateUserInput(client *kubernetes.Clientset, namespace string, svm string, shareToCheck string) error {
+	// Retrieve the users existing-cm
+	klog.Infof("Validating that user did not submit duplicate request " + namespace)
+	existingSharesCM, err := client.CoreV1().ConfigMaps(namespace).Get(context.Background(), existingSharesConfigMapName, metav1.GetOptions{})
```

Comment:
Although seriously doubt it will be common. I think we should extend this to also look at the requesting CM. Like, if they would have somehow requested the same thing twice, and the controller hadn't had time to run yet to process the first time they added that value.

We probably dont need to do a Get() call since we already have that "s3BucketsList" variable. And for that one, it would probably be more to check if the value is in s3BucketsList more than once.